### PR TITLE
Update to rails v5.2.0

### PR DESCRIPTION
**Pull request by neumayr**
* update to rails v5.2.0
  * `bundle update rails`
  * http://railsdiff.org/5.1.6/5.2.0
* setup `RAILS_MASTER_KEY` in heroku config
* update simple_form bootstrap v4 template
  * `rails generate simple_form:install --bootstrap`
* emojify footer 🍻

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,21 +1,18 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-  "https://github.com/#{repo_name}.git"
-end
-
+ruby '2.5.1'
```
Comment (klausmeyer) [inline Gemfile]: We already have this line down in the custom section.
Let's kill line 64 then and keep the default one up here.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require rails-ujs
+// require activestorage
```
Comment (klausmeyer) [inline app/assets/javascripts/application.js]: I think you're missing a `=` here?

File: app/views/application/_footer.html.erb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <div class="my-5">
-  <p class="text-center text-muted small"><i class="fa fa-beer"></i> Handcrafted in Bavaria.</p>
+  <p class="text-center text-muted small">🍺 Handcrafted in Bavaria. 🥨</p>
```
Comment (klausmeyer) [inline app/views/application/_footer.html.erb]: Beer & Pretzels. Jawoll 🦁

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (neumayr) [reply app/assets/javascripts/application.js]: left it out intentionally, [`active_storage/engine`](https://github.com/fidor/fidor_api_demo/pull/34/files#diff-b1fe55db50c712fef0673345e5b9c0d9R8) isn't load in this app.

Comment (klausmeyer) [reply app/assets/javascripts/application.js]: :+1:

**Pull request merged**